extensions/civetweb/processors/ListenHTTP.h

```diff
@@ -39,15 +39,21 @@
 #include "utils/gsl.h"
 #include "utils/Export.h"
 #include "utils/RegexUtils.h"
+#include "core/FlowFileStore.h"
+
+namespace org::apache::nifi::minifi::test {
+struct ListenHTTPTestAccessor;
+}  // namespace org::apache::nifi::minifi::test
 
 namespace org::apache::nifi::minifi::processors {
 
 class ListenHTTP : public core::Processor {
  private:
-  static constexpr std::string_view DEFAULT_BUFFER_SIZE_STR = "20000";
+  static constexpr std::string_view DEFAULT_BUFFER_SIZE_STR = "5";
+  static const core::Relationship Self;
```

Comment:
```suggestion
  static constexpr core::RelationshipDefinition Self{"__self__", "Marks the FlowFile to be owned by this processor"};
```

Reply:
I committed this in [4ab6514](https://github.com/apache/nifi-minifi-cpp/pull/1826/commits/4ab6514baf370199eb515c3816218d46b002261f). Feel free to remove it if you don't like the change.